.github/dependabot.yml

```diff
@@ -0,0 +1,10 @@
+version: 2
+updates:
+ - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "monthly"
```

Comment:
Should we change to this to weekly as mentioned [here](https://github.com/langchain-ai/langchain-aws/pull/346#discussion_r1941819410), to better cover our typical release cadence? 

Bi-weekly would be optimal - this will be possible in the near future, as DependaBot is adding support for cron jobs soon: https://github.com/dependabot/dependabot-core/issues/6339#issuecomment-2628628530